### DISCUSSION TITLE
How to prompt user with confirmation before paste multiple lines of text in clipboard

### DISCUSSION THREAD

**Discussion by snowman** [wezterm/wezterm#5414] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

It's okay to paste single line of text from clipboard when right click, because we can delete it if it's by mistake, so it will NOT run the pasted text immediately.

But sometimes accidental right-click may paste multiple lines of text to wezterm without notice, and it run every line of text which may not be the code, results in many garbage output

**Comment by symphorien** [comment]
xfce$-terminal has this feature (show unsafe paste dialog in the screenshot of the general tab here https://docs.xfce.org/apps/xfce4-terminal/preferences )
